applications/nrf5340_audio/broadcast_source/main.c

```diff
@@ -407,7 +406,11 @@ static int ext_adv_populate(uint8_t big_index, struct broadcast_source_ext_adv_d
 		return ret;
 	}
 
-	ret = broadcast_source_ext_adv_populate(big_index, ext_adv_data,
+	bool fixed_id = !IS_ENABLED(CONFIG_BT_AUDIO_USE_BROADCAST_ID_RANDOM);
```

Comment:
```suggestion
	bool use_fixed_id = !IS_ENABLED(CONFIG_BT_AUDIO_USE_BROADCAST_ID_RANDOM);
```
I think the code is more readable if it's clearer that this is a flag - especially considering it has the same format as `broadcast_id` which is the actual number. I think the name `use_fixed_id` could be used in the function headers as well

Reply:
Or `broadcast_id_fixed`?

Reply:
Resolved in offline discussion